.travis.yml

```diff
@@ -1,29 +1,37 @@
 language: rust
 dist: trusty
 sudo: required
-rust:
-- beta
-- nightly
-- stable
 
 cache:
   directories:
   - target
 
-os:
-  - linux
-#  - osx
+
+matrix:
+  include:
+    - os: linux
+      rust: nightly
+      env: SKIP_FORMAT_CHECK=false
+    - os: linux
+      rust: beta
+      env: SKIP_FORMAT_CHECK=true
+    - os: linux
+      rust: stable
+      env: SKIP_FORMAT_CHECK=true
```

Comment:
I think you can disable format check in all travis ci, since circle ci has checked it already.